README.md

```diff
@@ -35,183 +27,57 @@ OpenTTD is licensed under the GNU General Public License version 2.0,
-parameter or by typing '`debug_level desync=<level>`' in OpenTTD's internal
-console.
-The desync debug levels are:
-
-- 0: nothing.
-- 1: dumping of commands to 'commands-out.log'.
-- 2: same as 1 plus checking vehicle caches and dumping that too.
-- 3: same as 2 plus monthly saves in autosave.
-- 4 and higher: same as 3
-
-Restarting OpenTTD will overwrite 'commands-out.log'. OpenTTD will not remove
-the savegames (dmp_cmds_*.sav) made by the desync debugging system, so you
-have to occasionally remove them yourself!
-
-The naming format of the desync savegames is as follows:
-dmp_cmds_XXXXXXXX_YYYYYYYY.sav. The XXXXXXXX is the hexadecimal representation
-of the generation seed of the game and YYYYYYYY is the hexadecimal
-representation of the date of the game. This sorts the savegames by game and
-then by date making it easier to find the right savegames.
-
-When a desync has occurred with the desync debugging turned on you should file
-a bug report with the following files attached:
-
-- commands-out.log as it contains all the commands that were done
-- the last saved savegame (search for the last line beginning with
-   'save: dmp_cmds_' in commands-out.log). We use this savegame to check
-   whether we can quickly reproduce the desync. Otherwise we will need …
-- the first saved savegame (search for the first line beginning with 'save'
-   where the first part, up to the last underscore '_', is the same). We need
-   this savegame to be able to reproduce the bug when the last savegame is not
-   old enough. If you loaded a scenario or savegame you need to attach that.
-- optionally you can attach the savegames from around 50%, 75%, 85%, 90% and
-   95% of the game's progression. We can use these savegames to speed up the
-   reproduction of the desync, but we should be able to reproduce these
-   savegames based on the first savegame and commands-out.log.
-- in case you use any NewGRFs you should attach the ones you used unless
-   we can easily find them ourselves via bananas or when they are in the
-   #openttdcoop pack.
-
-Do NOT remove the dmp_cmds savegames of a desync you have reported until the
-desync has been fixed; if you, by accident, send us the wrong savegames we
-will not be able to reproduce the desync and thus will be unable to fix it.
-
-## 3.0) Supported platforms
-
-OpenTTD has been ported to several platforms and operating systems. It should
-not be very difficult to port it to a new platform. The currently working
-platforms are:
+## 1.1) Downloading OpenTTD
+
+OpenTTD can be downloaded from the [official OpenTTD website](https://www.openttd.org/).
+
+Both 'stable' and 'nightly' versions are available for download.
+
+Most people will want the 'stable' version.
+
+The 'nightly' version includes the latest code changes and may be less reliable.
+
+On some platforms OpenTTD will also be available via your OS package manager or a similar service.
```

Comment:
this section reads a bit clumsy, i don't really like 1-line-paragraphs.

Reply:
Me neither, but it's house style, and irc logs show it's been mentioned a few times as a requirement.

Reply:
No, you completely missed how that was meant.

it was meant that different sentences **in the same paragraph** should be on separate lines. not that each sentence should be a separate paragraph.

Reply:
Reformatted most of the doc.  The argument about isolated sentences vs. paragraphs could go on ad-infinitum.